src/main/java/de/zalando/aruha/nakadi/domain/Cursor.java

```diff
@@ -0,0 +1,52 @@
+package de.zalando.aruha.nakadi.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public class Cursor {
+
+    private final String partition;
+
+    private final String offset;
+
+    public Cursor(@JsonProperty("partition") final String partition, @JsonProperty("offset") final String offset) {
+        this.partition = partition;
+        this.offset = offset;
+    }
+
+    public String getPartition() {
+        return partition;
+    }
+
+    public String getOffset() {
+        return offset;
+    }
+
+    @Override
+    public boolean equals(final Object o) {
```

Comment:
I've added a dependency that might make this easier to implement. Maybe you would like to use it here, just like this https://github.com/zalando/nakadi/blob/nakadi-jvm/src/main/java/de/zalando/aruha/nakadi/domain/EventTypeSchema.java#L35-L37